srcpkgs/batsignal/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'batsignal'
+pkgname=batsignal
+version=1.1.2
+revision=1
+archs=noarch
+build_style=gnu-makefile
+makedepends=" libnotify-devel make pkg-config glib-devel "
+short_desc="Simple battery monitor"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
+license="ISC"
+homepage="https://github.com/electrickite/batsignal"
+distfiles="https://github.com/electrickite/batsignal/archive/${version}.tar.gz"
+checksum=00cb30df33b5fee049963657f341e963a0f094dbf1620397bf1e56fa07191e7c
+
+do_build() {
+	xmake PREFIX=/usr DESTDIR="${pkgdir}" install
+	vlicense LICENSE
+}
```

Comment:
`xmake` doesn't exist in void.

Since you're using the `gnu-makefile` build style, first try it without setting any function, its defaults are usually good enough for most programs. If it works, you will only need to run `vlicense LICENSE` in `post_install()`.

Reply:
Actually that's a typo. And now magically appears fails the compilation because of glib.h ... 
Isn't that on the glib-devel ?

Reply:
It's in `libglib-devel`, but that's a `glib-devel` dependency anyway, so it should be available.

Note that most builds don't require runnning `make <command>` manually, you should leave it up to the build style. If, however, you need to run `make install` manually for some reason that isn't covered by setting `make_install_args`, you should run `make install` inside `do_install`, and `DESTDIR` should be set to `$DESTDIR`.

Reply:
Fixed that thing ! But now I keep getting that glib.h , I am putting the dependencies on the wrong side ?